src/examples/example_contract.rs

```diff
@@ -59,7 +61,7 @@ fn main() {
     let wasm = wabt::wat2wasm(CONTRACT).expect("invalid wabt");
 
     // 1. Put the contract code as a wasm blob on the chain
-    let xt = contract::put_code(api.clone(), 500_000, wasm);
+    let xt = api.put_code(500_000, wasm);
```

Comment:
I don't like the namespacing: put_code and create refer to contract module only but the get a top-level function in the api. I'd prefer `api.contract.put_code()` and `api.contract.create()`

Reply:
I have been trying to get my head arround this. We could do it like: https://github.com/paritytech/substrate-subxt/blob/master/src/srml/balances.rs

But I am not sure if this is unnecesarily complex. Maybe just revert it to the way it was?

Reply:
My suggestion is only about the naming, not the general approach. It could also be a prefix like àpi.balances_transfer()` and `api.contract_put_code()`. A touch less elegant though.

Reply:
let's leave this for a refactoring task. I don't want to keep this PR stuck because of a detail